### ISSUE TITLE
Use CPS Index Image Large where Media imageURL is null 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#8254]
**Is your feature request related to a problem? Please describe.**
The media data in our ares assets returns an imageURL of null occasionally where MPV cannot process the data
When that happens we want to render an image rather than showing no image in the player at all 
See https://www.bbc.com/mundo/media-54661776 (and image attached) 

![Screenshot 2020-10-27 at 16 12 55](https://user-images.githubusercontent.com/22519682/97329789-5b789e00-186f-11eb-9a2a-44567f8c97b8.png)

**Describe the solution you'd like**
Given media imageUrl is null e.g. `"imageUrl": null,` within the ARES media block in a MAP asset
then use `promos.indexImage.href` in the player instead 

**Media block with image null**

```
blocks": [
{
"advertising": true,
"caption": "¡Esto es BBC Minute en español!",
"embedding": true,
"format": "audio",
"id": "p08wkvzp",
"imageUrl": null,
"smpKind": "radioProgramme",
"subType": "clip",
"synopses": {
"long": "Cinco historias, 60 segundos. Esto es BBC Minute, deportes, tendencias, música, medio ambiente… En un minuto Beatriz De La Pava y Alba Morgade te cuentan lo que es noticia en el mundo",
"medium": "Un minuto con las noticias más importantes y compartidas del mundo.",
"short": "¡Esto es BBC Minute en español!"
},
"title": "BBC Minute - 1500 GMT - 27.10.20",
"type": "media",
"versions": [
{
"availableFrom": 1603810329000,
"availableTerritories": {
"nonUk": true,
"uk": true
},
"availableUntil": 1603983129000,
"duration": 60,
"durationISO8601": "PT1M",
"types": [
"Original"
],
"versionId": "p08wkvzr",
"warnings": {}
}
]
},
{
"markupType": "plain_text",
"text": "Cinco historias, 60 segundos. Esto es BBC Minute, deportes, tendencias, música, medio ambiente… En un minuto Beatriz De La Pava y Alba Morgade te cuentan lo que es noticia en el mundo",
"type": "paragraph"
}
]
},
```

**Index Image**
```
"promo": {
"headlines": {
"headline": "¡Esto es BBC Minute en español!",
"shortHeadline": "BBC Minute, escucha las noticias del día en 60 segundos"
},
"id": "urn:bbc:ares::asset:mundo/media-54661776",
"indexImage": {
"altText": "BBC Minute con Beatriz y Jose",
"copyrightHolder": "BBC",
"height": 549,
"href": "http://c.files.bbci.co.uk/38D5/production/_108194541_promopic.jpg",
"id": "108194541",
"path": "/cpsprodpb/38D5/production/_108194541_promopic.jpg",
"subType": "index",
"type": "image",
"width": 976
},
```

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by aeroplanejane** [comment]
This requires a quick chat with test and me before work commences on this issue. 
We will only be able to test this by changing fixture data locally - we do not have test or live scenarios for this (with the exeption of the asset linked in the ticket) 
Example attached: 
[mundo:media-54661776.txt](https://github.com/bbc/simorgh/files/5446624/mundo.media-54661776.txt)

**Comment by rebeccamcginn** [comment]
Closed by automation; opening for testing

**Comment by LilyL0u** [comment]
I am having problems testing this issue. I am changing the fixture data so the ImageUrl is null, but the image that was there before still shows. I have removed all image urls in the whole fixture data just in case it was coming from somewhere else, but I cannot make the image disappear, and so cannot replicate the issue. Where is the image coming from in this case if information has been removed about it from the fixture data? From CPS/Castaway?
@pharingee any ideas?
